code/script.js

```diff
@@ -1,343 +1,557 @@
-          <span>Guess on ${person.name}?</span>
-          <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
+      <div id="card" class="card">
+        <div class="card-inner">
+          <div class="card-front">
+            <p>${person.name}</p>
+            <div class="card-img">
+              <img src=${person.img} alt=${person.name} />
+            </div>
+          </div>
+          <div class="card-back">
+            <div class="guess">
+              <span>
+                Guess on ${person.name}?
+              </span>
+              <button id ="guess-btn" class="filled-button small" onclick="guess('${person.name}')">
+                Guess
+              </button>
+            </div>
+          </div>
         </div>
       </div>
     `
   })
+
+  const counter = document.getElementById("question-counter")
+  const card = document.querySelectorAll(".card-front")
+  counter.innerHTML = `Questions asked: ${questionsAsked}` //displays how many questions a player asked
+  cardFlipAudio(card) //sound on hover when cards are flipped
+}
+
+//adds sound to element
+const cardFlipAudio = (card) => {
+  const audio = document.getElementById("card-sound")
+  card.forEach(card => card.addEventListener("mouseenter", () => audio.play()))
+  card.forEach(card => card.addEventListener("mouseout", () => audio.play()))
 }
 
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
+//timer showing the passing minutes/seconds in the game
+const timer = (x) => {
+  if (x === "restart") {
+    totalSeconds = 0;
+  }
+  totalSeconds++
+  let minute = Math.floor(totalSeconds / 60);
+  let seconds = totalSeconds - (minute * 60);
+  if (minute < 10) {
+    minute = "0" + minute;
+  }
+  if (seconds < 10) {
+    seconds = "0" + seconds;
+  }
+  countUp.innerHTML = `Time passed: ${minute}:${seconds}`;
+}
+
+//stops the timer and resets time to 0
+const stopTimer = () => {
+  let setTimer = setInterval(timer, 1000);
+  clearInterval(setTimer)
+  timer("restart")
+}
+
+//make it possible for the player to add name before starting the game
+const preGame = () => {
+  const form = document.getElementById("form");
+  form.classList.add("active") //shows the form section
+
+  board.innerHTML = ""
+  form.innerHTML += `
+    <label for="name-input">
+      Insert your name:
+    </label>
+    <input id="name-input" name="name-input" type="text"/>
+    <button id="name-Btn" class="filled-button">
+      Let's play!
+    </button>
+  `
+  const nameBtn = document.getElementById("name-Btn");
+  const userName = document.getElementById("name-input")
+  nameBtn.addEventListener("click", () => {
+    player = userName.value; //stores nameinput
+    form.innerHTML = ""
+    form.classList.remove("active") //hides form with nameinput
+    start() //runs start
+  })
```

Comment:
I see that you both added and removed the form in the same function. I think it was a really good idea since now nothing else has to anyting "to do" with what happens inside. I can imagine it would be easier to have issues if you would have to start another function with removing the form.